src/pages/login/index.tsx

```diff
@@ -0,0 +1,92 @@
+import InputBox from '@/components/InputBox';
+import Layout from '@/components/Layout';
+import { ChangeEvent, SetStateAction, useState } from 'react';
+
+const LoginPage = () => {
+    const [usernameBorder, setUsernameBorder] = useState<SetStateAction<string>>('');
+
+    const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+        const inputValue = (event.target as HTMLInputElement).value;
+        const regex = /^[a-zA-Z0-9_]+$/;
```

Comment:
- Should we move this to a constants file?

Reply:
+1

Reply:
It would be great attaching a ticket here